.vscode/settings.json

```diff
@@ -0,0 +1,11 @@
+{
+    "files.exclude": {
+        "**/.git": true,
+        "**/.svn": true,
+        "**/.hg": true,
+        "**/CVS": true,
+        "**/.DS_Store": true,
+        "**/Thumbs.db": true,
+        "**/node_modules": true
+    }
+}
```

Comment:
I'm not sure if VSCode settings should be committed. Doesn't seem like anything important, only adds another folder